libs/common/src/tools/generator/index.ts

```
@@ -0,0 +1,2 @@
+export * from "./password";
+export * from "./username";
```

Seems we rarely if ever use this barrel file? Should the imports be updated to use `generator` instead of `generator/password`? We should also forbid direct imports of anything nested below, (and temporarily ignore errors for the components we have to expose)

---

You are right, the barrel-file was unnecessary. I've removed it. Albeit a longer import path, it is better to be specific about an import.